src/main/java/com/dzionek/simpleshell/commands/nonaffecting/displaying/DirCommand.java

```diff
@@ -0,0 +1,33 @@
+package com.dzionek.simpleshell.commands.nonaffecting.displaying;
+
+import com.dzionek.simpleshell.commands.nonaffecting.BehaviorCommand;
+import com.dzionek.simpleshell.shellmanaging.ShellData;
+
+import java.io.File;
+
+/**
+ * @author Bartlomiej Janik
+ * @since 9/16/2017
+ */
+public class DirCommand implements BehaviorCommand {
+
+
+    public void executeBehavior(ShellData shellData) {
+        StringBuilder stringBuilder = new StringBuilder();
+        File[] filesInDirectory = new File(shellData.getCurrentDirectory()).listFiles();
+        for (int i = 0; i < filesInDirectory.length - 1; ++i) {
```

Comment:
czemu nie for-each ? a jeszcze lepiej na lambdzie :P bo masz array

Reply:
ale czytelne, tylko for-each lepszy bo nie musisz używać jawnie "i", a jak ktoś lubi funkcyjnie to stream